.travis.yml

```diff
@@ -3,19 +3,30 @@ os:
   - linux
   - osx
 julia:
-  - 0.6
+  - 0.7
+  - 1.0
```

Comment:
@Evizero: If you do this, you also need to bump REQUIRE to 0.7. I recommend that since a lot of packages drop 0.6 support on master. (@mbauman for attention)

Reply:
right